### DISCUSSION TITLE
LAN Server Problems

### DISCUSSION THREAD

**Discussion by ghost** [ldtteam/minecolonies#8538] [General]
Hello, i´m not sure if it´s related to MineColonies, but the problem occurs since we installed it.

So, i´m playing with my girlfriend on my Singelplayer world using "Open to LAN" which always worked fine, provided we both got the same forge and mod versions.
Yesterday my girlfriend got kicked from the server with "Internal Exception: java.lang.IndexOutOfBoundsException"
Just kicked from the server, minecraft didn´t crash, therefore i can´t find any crash report on her pc. Nothing at all.

I kept the minecraft launcher console running in the background on both pc´s and there is also no much to work with...
I´ve no clue what´s going on. If i could only get a more specific error or a crash log...

Any suggestions :/ ?

**Comment by Nightenom** [comment]
When you get error and do not crash you can still get debug/latest logs from both sides (ie. client + server or here 2 clients), they will for sure contain the error you mentioned
